WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/ui/preferences/PreferencesBase.java

```diff
@@ -0,0 +1,47 @@
+package nl.tudelft.watchdog.core.ui.preferences;
+
+import java.util.List;
+
+public abstract class PreferencesBase {
```

Comment:
At first, it was an interface, but I had some class loading issues with it. I think they are resolved now, so I will put interface, thanks!